aten/src/ATen/native/cpu/SoftMaxKernel.cpp

```diff
@@ -42,10 +42,11 @@ inline void _vec_log_softmax_lastdim(
   // size of L1D cache on many processors. Some processors have 48 KB L1D cache
   // nowadays, so maybe in the future, we can leverage the knowledge of a
   // machine's L1D cache size.
-  int64_t CHUNK_SIZE = std::max<int64_t>(
+  int64_t MAX_CHUNK_SIZE = std::max<int64_t>(
       1,
       at::internal::GRAIN_SIZE / (sizeof(scalar_t) * dim_size));
-
+  int64_t CHUNK_SIZE = std::min<int64_t>(MAX_CHUNK_SIZE, outer_size);
```

Comment:
Can you please also check its applicability to other kernels in the same file?
IIRC, Jiong had asked me to use `CHUNK_SIZE` from one of the kernels below, so it's likely that they may also benefit.

Also, can you please add benchmarks to `benchmarks/operator_benchmark/pt/softmax_test.py`, if they don't already exist? And can you please add benchmarking data for more practical shapes? The benchmarks in those files are obsolete. Thanks!

Reply:
Marking as resolved because @min-jean-cho mentioned she'd add benchmarks later